kotlin/services/glue/Readme.md

```diff
@@ -1,29 +1,127 @@
 # AWS Glue Kotlin code examples
```

Comment:
AWS Glue code examples for the SDK for Kotlin

Reply:
fixed